partridge/readers.py

```diff
@@ -1,20 +1,18 @@
-from collections import defaultdict
```

Comment:
Please keep imports in alphabetical order. It looks like the only import related change in this file should be the removal of `from .utilities import remove_node_attributes`.